doc/api/cli.md

```diff
@@ -1049,6 +1049,16 @@ added:
 
 Use this flag to enable [ShadowRealm][] support.
 
+### `--experimental-strip-types`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.0 - Early development
+
+Enable experimental type-stripping for typescript files.
```

Comment:
IMO this shouldn't say 'TypeScript files', as it would technically strip the types on any file?